test/ngAria/ariaSpec.js

```diff
@@ -414,49 +423,24 @@ describe('$aria', function() {
   describe('aria-required', function() {
```

Comment:
I don't think `aria-required` has to be removed if `ngRequired` is false, just set to `aria-required="false"`. It isn't a boolean attribute like `required`. I will add a test for that, though. https://www.w3.org/TR/wai-aria/states_and_properties#aria-required